integration-test/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/docker/SparkDockerImageBuilder.scala

```diff
@@ -64,6 +65,7 @@ private[spark] class SparkDockerImageBuilder
     buildImage("spark-base", BASE_DOCKER_FILE)
     buildImage("spark-driver", DRIVER_DOCKER_FILE)
     buildImage("spark-executor", EXECUTOR_DOCKER_FILE)
+    buildImage("spark-init-container", INIT_CONTAINER_DOCKER_FILE)
```

Comment:
Please also add an option to allow the test to directly use an init container supplied externally.